library/src/main/java/com/novoda/downloadmanager/lib/StorageManager.java

```diff
@@ -41,52 +42,61 @@
     /**
      * the max amount of space allowed to be taken up by the downloads data dir
      */
-    private static final long sMaxdownloadDataDirSize = 100 * 1024 * 1024;
+    private static final long MAX_DOWNLOAD_DATA_DIR_SIZE = 100 * 1024 * 1024;
```

Comment:
Worth specifying the unit as well in the name `MAX_DOWNLOAD_DATA_DIR_SIZE_BYTES`?